### PR TITLE
feat(route): 北极星光伏

### PR DESCRIPTION
**Pull request by 1065464173**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/bjx/gf/sc
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [ ] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: assets/radar-rules.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 ({
+    'guangfu.bjx.com.cn': { _name: '北极星光伏', '.': [{ title: '分类', docs: 'https://docs.rsshub.app/others.html#bei-ji-xing-guang-fu', source: ['/:type', '/'], target: '/bjxgf/:type?' }] },
```
Comment (TonyRL) [inline assets/radar-rules.js:2]: Do not add radar rules here.
You should add the rules in `lib/v2/bjxgf/radar.js`

File: docs/other.md [hunk 2]
```diff
@@ -4,6 +4,18 @@ pageClass: routes
 
 # 其他
 
+## 北极星光伏
+
```
Comment (TonyRL) [inline docs/other.md:8]: Missing H3 heading

**Review by TonyRL — CHANGES_REQUESTED** [review]
Missing `lib/v2/bjxgf/router.js`

**Comment by TonyRL** [comment]
I would recommend using `bjx` instead of `bjxgf`

**Comment by 1065464173** [comment]
> Missing `lib/v2/bjxgf/router.js`

Found it works without `lib/v2/bjxgf/router.js`,amazing!!

> I would recommend using `bjx` instead of `bjxgf`

Thinks  for your advice.  finding it already exists in `lib/routers/` by others.
Should I put it together in v2 ...?? （Stupidly asked）

--- 
I'll push it in right rules tomorrow！

**Comment by TonyRL** [comment]
> Thinks for your advice. finding it already exists in `lib/routers/` by others. Should I put it together in v2 ...?? （Stupidly asked）
> 
You can put it together if you want. If you don't, I will do it before merging.

**Comment by fucao** [comment]
想要的是北极星风电网，不是光伏网

**Pull request merged**